source/configuration_manual/mail_location/Maildir/index.rst

```diff
@@ -141,6 +141,35 @@ specified, we get an extra path component ``INBOX/`` immediately prior to the
 
 The value for ``DIRNAME`` should be chosen carefully so as to minimise the chances of clashing with mail folder names. In the example here, unusual upper/lower casing has been used.
 
+Multiple Namespaces pointing to INBOX
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When there are multiple namespaces that point to the same INBOX namespace,
+``dovecot.list.index`` can potentially keep fighting over whether INBOX exists
+or not.
+
+For example::
```

Comment:
This(`::`) is nice and compact compared to ``.. code-block::none`` but it defaults to highlight the code as python which makes colons and brackets weirdly big when this is getting rendered.